backEnd/tests/test_hashing.py

```diff
@@ -0,0 +1,42 @@
+import pytest
+from ..security.hashing import hash_string, verify_password
+
+def test_hash_string():
```

Comment:
```suggestion
def hash_string_mustReturnTheHashedString_WhenInputIsInvalid():
```
This naming convention follows best practices for test function names. It helps ensure that, when running unit tests, we can easily identify the specific functionality that has failed based on the test output.

This is a suggestion for future work, but please consider updating the test function names when you have the opportunity.